checker/definitions/overrides.d.ts

```diff
@@ -201,6 +201,29 @@ declare class String {
 
 declare class Promise<T> { }
 
+declare class RegExp {
+    @Constant("regexp:constructor")
+    constructor(pattern: string, flags?: string);
+
+    @Constant("regexp:exec")
+    exec(input: string): RegExpExecArray | null;
+}
+
+interface RegExpExecArray extends Array<string> {
```

Comment:
Man JS spec is crazy. So the returned result is an array with a bunch of added string/named properties! I think this definition where an interface extends a class works 🤞